src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt

```diff
@@ -70,6 +70,7 @@ class SnykApplicationSettingsStateService : PersistentStateComponent<SnykApplica
         XmlSerializerUtil.copyBean(state, this)
     }
 
+    @Suppress("DEPRECATION")
```

Comment:
shell it be down the code, close to actually deprecated val?

Reply:
I think it should be on method level, else we have more suppression lines than actual code

Reply:
hmm, just one suppression cover all code in the method below. So moving it two line down do the job the same way as it now but bring it close to first deprecated val usage appearance
PS minor, does not really matter

Reply:
We would need it two times - once for the `if` and once for the assignment